pkg/datapath/linux/node.go

```diff
@@ -571,19 +571,19 @@ func (n *linuxNodeHandler) insertNeighbor(newNode *nodeTypes.Node, ifaceName str
 
 	iface, err := net.InterfaceByName(ifaceName)
 	if err != nil {
-		neighborLog("insertNeightbor InterfaceByName", ifaceName, err, &ciliumIPv4, &hwAddr, link)
+		neighborLog("insertNeighbor InterfaceByName", ifaceName, err, &ciliumIPv4, &hwAddr, link)
```

Comment:
:100: 

nits: I feel it's kind of repetitive to mention inserNeighbor here as underlying _neighborLog_ is already log this text. https://github.com/cilium/cilium/blob/master/pkg/datapath/linux/node.go#L546

Reply:
Good point. I think we should just drop the `insertNeighbor` part from the messages passed to `neighborLog`.

Reply:
Agree with you, good comments, Thanks.